app/views/RoomsListView/styles.ts

```diff
@@ -7,7 +7,8 @@ export default StyleSheet.create({
 		flex: 1
 	},
 	list: {
-		width: '100%'
+		width: '100%',
+		flex: 1
```

Comment:
I tested with multiple devices, but I don't know why there wasn't the `flex: 1` before. I added this flex, otherwise, I needed to add the `flex: 1` as style at KeyboardView, but this introduced a long padding bottom as you can see on this video:

https://user-images.githubusercontent.com/47038980/223264243-26280e17-5983-43ba-b7e7-3a95b28f5deb.mov